### PR TITLE
Change Kyma CLI version to 2.2.0 in Compass scripts

### PR DESCRIPTION
**Pull request by nyordanoff**
**Description**

Basically the title.

Changes proposed in this pull request:

- change Kyma CLI version

**Related issue(s)**

### CODE REVIEW

File: prow/scripts/cluster-integration/compass-gke-benchmark.sh [hunk 1]
```diff
@@ -139,7 +139,7 @@ function installHelm() {
 }
 
 function installKyma() {
-  KYMA_CLI_VERSION="2.1.3"
+  KYMA_CLI_VERSION="2.2.0"
```
Comment (Sawthis) [inline prow/scripts/cluster-integration/compass-gke-benchmark.sh:142]: Why do you use so old version of Kyma CLI? Kyma CLI 2.9.1 was already released.

File: prow/scripts/provision-vm-compass.sh [hunk 2]
```diff
@@ -186,7 +186,7 @@ if [[ -z "${VM_FOR_PREFIX_AND_SUFFIX}" ]]; then
     utils::compress_send_to_vm "${ZONE}" "${VM_PREFIX}${SUFFIX}" "/home/prow/go/src/github.com/kyma-incubator/compass" "~/compass"
 
 
-    KYMA_CLI_VERSION="2.1.3"
+    KYMA_CLI_VERSION="2.2.0"
```
Comment (Sawthis) [inline prow/scripts/provision-vm-compass.sh:189]: Why do you use so old version of Kyma CLI? Kyma CLI 2.9.1 was already released.

**Review by Sawthis — CHANGES_REQUESTED** [review]

File: prow/scripts/cluster-integration/compass-gke-benchmark.sh (hunk 1 above)
Comment (nyordanoff) [reply prow/scripts/cluster-integration/compass-gke-benchmark.sh:142]: That's the current version we are adopting in Compass. We have regular tasks to adopt new Kyma versions but it's not a trivial process.

File: prow/scripts/provision-vm-compass.sh (hunk 2 above)
Comment (nyordanoff) [reply prow/scripts/provision-vm-compass.sh:189]: That's the current version we are adopting in Compass. We have regular tasks to adopt new Kyma versions but it's not a trivial process.

**Review by Sawthis — APPROVED** [review]

**Pull request merged**